test/validation/api/atomic/atomic.c

```diff
@@ -1124,6 +1182,15 @@ static int test_atomic_max_thread(void *arg UNUSED)
 	return CU_get_number_of_failures();
 }
 
+static int test_atomic_fetch_max_thread(void *arg UNUSED)
+{
+	thread_init();
+	test_atomic_fetch_max_32();
+	test_atomic_fetch_max_64();
+
+	return CU_get_number_of_failures();
```

Comment:
This returns not only the number of failures seen in this test but the total so far, as is done by the existing functions too. This is not very nice since a test failure causes printouts from all the subsequent tests too, even if they pass.

Reply:
True, removed in V3. Will clean the existing tests in a separate PR.